examples/is31fl3731/main.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"time"
+
+	"machine"
+
+	"tinygo.org/x/drivers/is31fl3731"
+)
+
+// I2CAddress -- address of led matrix
+var I2CAddress uint8 = 0x74
```

Comment:
How random is this address? Can it be moved to the driver itself?

Reply:
For standalone is31fl3731 the addresses can be:
- 0x74 (AD pin connected to GND)
- 0x75 (AD pin connected to SCL)
- 0x76 (AD pin connected to SDA)
- 0x77 (AD pin connected to VCC)

For Adafruit Charlie Wing 15x7 its either 0x74 or 0x77. I guess I can make four constants, but it's up to the particular board to define address. WDYT?

Reply:
I'd say have them 4 constants in the driver "registers.go" as per praxis and seen in other drivers.
Preferably with comments (exactly as you did here in comment).
And let user of the driver choose which address to use.

Reply:
Disclaimer, I'm not a maintainer, but submitted several drivers already to this repo and sort of know the drill.
I like your PR and just want to help it :)

Reply:
got it, thanks for the help! :)

Reply:
Added address constants 👍